main.py

```diff
@@ -71,7 +71,16 @@ class HomePageRedirectHandler(base.BaseHandler):
     """
     def get(self):
         if self.user_id and user_services.has_fully_registered(self.user_id):
-            self.redirect(feconf.DASHBOARD_URL)
+            # Get all the explorations associated with the user's account
```

Comment:
> write backend tests for this in order to verify the behaviour. You can do so in base_test.py.

@seanlip  any links, documentation on how to do it. I am a newbie in these kind of stuff. I can learn fast if I can find a starting point on how to do it.